### DISCUSSION TITLE
GH Pages understands /favicon.ico as a route

### DISCUSSION THREAD

**Discussion by thiagosaud** [community/community#102085] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm having a problem with the `favicon.ico` in search engines, it doesn't appear, I also noticed that when I type `/favicon.icon `on my site, it understands it as a route, as in `404.html` I add the code below, it redirects to `index.html`.

`<meta http-equiv="refresh" content="0; URL=https://thiagosauddev.github.io/" />`

What should I do to stop GH pages from understanding that `/favicon.icon `is not a non-existent route?

MY PROJECT CODE
https://github.com/thiagosauddev/thiagosauddev.github.io

**Comment by yashmalvade2002** [comment]
ask chatgpt

**Comment by yoannchaudet** [comment]
Your icon is set to a different URL:

<img width="751" alt="image" src="https://github.com/community/community/assets/14911070/34c9d984-8a18-45aa-861f-6627a203cc98">

I don't see anything wrong with your configuration and I am not sure what you are asking for with regard to search engines.

**Reply by thiagosaud** [reply]
I also don't see anything wrong with my setup, but search engines don't display the favicon, look:

![image](https://github.com/community/community/assets/126628718/b8f9b9f8-fc81-4ea1-a374-fc9e856de948)

**Reply by yoannchaudet** [reply]
Refer to the search engine's doc maybe. Google for instance publishes some guidelines here: https://developers.google.com/search/docs/appearance/favicon-in-search
